### PR TITLE
[sailjail-permissions] Add audioroute service to Audio permissions. Contributes to JB#52175

### PR DESCRIPTION
**Pull request by pvuorela**
Needed by fm radio support.

@rainemak @Tomin1

### CODE REVIEW

**Comment by rainemak** [comment]
Please use "Fixes JB#52175"

**Comment by pvuorela** [comment]
Was playing cautious in case there are still some device specific details, but can also change if you see better.

File: permissions/Audio.permission [hunk 1]
```diff
@@ -22,3 +22,7 @@ dbus-user.own org.mpris.MediaPlayer2.*
 dbus-system.talk org.maemo.resource.manager
 dbus-system.broadcast org.maemo.resource.manager=org.maemo.resource.manager.*@/*
 # END systembus-org.maemo.resource.manager.resource
+
+# BEG systembus-org.nemomobile.Route.Manager
+dbus-system.talk org.nemomobile.Route.Manager
```
Comment (rainemak) [inline permissions/Audio.permission:27]: No need for broadcasting?

**Review by rainemak — APPROVED** [review]
If this works, I'm happy. Approving.

**Pull request merged**